src/api-service/__app__/onefuzzlib/notifications/ado.py

```diff
@@ -175,6 +175,14 @@ def update_existing(self, item: WorkItem) -> None:
 
         if document:
             self.client.update_work_item(document, item.id, project=self.project)
+            logging.info(
+                "notify ado: updated work item" f" {item.id} - {notification_info}"
```

Comment:
How difficult is it to use a couple of customEvents for tracking when a work item is created and updated?

Reply:
as discussed. i will add it in a different PR